docs/calendar-draft.md

```diff
@@ -350,7 +351,39 @@ As compared to option 1 (always default to Full ISO), the following are examples
 
 These errors are most common in the four regions that use non-ISO calendars as their default: Saudi Arabia, Iran, Afghanistan, and Thailand.  However, they could also manifest if the user has overridden their preferred calendar in browser settings.
 
-For the purposes of illustration, the following examples will use the locale "en-SA" (English in Saudi Arabia) on 2020-05-18.
+For the purposes of illustration, the following examples will use the locale "en-SA" (English in Saudi Arabia) on 2020-05-17.
+
+#### Format a month and day from a date
+
+Buggy output:
+
+	Today is: May 17
+
+Correct output:
+
+	Today is: Ramadan 24
+
+Code:
+
+```javascript
+/// Options 1 and 4: calendar implicit
+const today = Temporal.now.date();
+// OPTION 1 BUG: the ISO calendar is implicit when creating the MonthDay
+const monthDay = today.getMonthDay();
+console.log("Today is:", monthDay.toLocaleString());
+
+/// Options 2 and 6: calendar in factory method
+const calendar = navigator.locales[0].getLikelyCalendar();
+const today = Temporal.now.date(calendar);
+const monthDay = today.getMonthDay();
+console.log("Today is:", monthDay.toLocaleString());
+
+/// Options 3 and 5: calendar only when needed
+const today = Temporal.now.date();
+const calendar = navigator.locales[0].getLikelyCalendar();
+const monthDay = today.getMonthDay(calendar);
```

Comment:
I'm not convinced this is a different use case than the below one, why would we add a calendar argument to `getMonthDay()` rather than calling `today.withCalendar(calendar).getMonthDay()` to convert `today` from partial to full ISO (3) or to get the appropriate calendared type (5)? Fundamentally this problem lies with `Temporal.now.date()`, not `getMonthDay()`.

Reply:
Makes sense.  I split this into two different explicit options.